browser/css/jsdialogs.css

```diff
@@ -541,6 +541,11 @@ td.jsdialog > [id^='table-box']:not(.sidebar) {
 	white-space: nowrap;
 }
 
+#ServerAuditDialog .ui-treeview-entry {
```

Comment:
This element should be set in a way that allows word wrapping as I said previously so that would be:

```css
white-space: normal;
```

Reply:
Added whitespace: normal ui-treeview-entry